loader/src/main/scala/com/snowplowanalytics/snowplow/storage/bigquery/loader/Loader.scala

```diff
@@ -67,6 +71,8 @@ object Loader {
       .flatMap {
         case (Right(row), ctx) =>
           ctx.output(ObservedTypesOutput, row.inventory)
+          val latency = (Instant.now().getMillis - row.collectorTstamp.getMillis) / 1000
```

Comment:
Are we sure seconds is enough? It seems its possible to make in miliseconds.

Reply:
It can be in milliseconds but I double-checked with Product and seconds is the unit of choice.